script/foreman-rake

```diff
@@ -10,7 +10,9 @@ die() {
 cd ~foreman || die "Cannot change to foreman home directory"
 [ -f Gemfile ] && BUNDLER_CMD="bundle exec"
 CMD="$BUNDLER_CMD $RAKE_CMD"
-if [ $USERNAME = foreman ]; then
+if [ $# -eq 0 ]; then
+  foreman-rake -h; exit 0;
```

Comment:
perfect, Thanks @domcleal for pointing out the issue. Will update the script with
$CMD -h; exit 0;